CONTRIBUTING.md

```diff
@@ -18,23 +20,176 @@ This project just started! Please wait for the first few versions to be released
 
 ## Responsibilities
 
-- Ensure cross-platform compatibility for every change that's accepted. Windows, Linux, Android, Xbox, iOS.
+- Ensure cross-platform compatibility for every change that's accepted. Windows, Linux, Android, iOS, Xbox, Playstation.
 - Ensure that code that goes into master branch meets all requirements from the requirements list below
 - Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
-- Don't add any classes to the codebase unless needed. Err on the side of using functions.
+- Don't add any classes to the codebase unless needed.
 - Keep feature versions as small as possible, preferably one new major feature per version.
 - Be welcoming to newcomers and encourage diverse new contributors from all backgrounds. See the [Contributor Covenant](https://www.contributor-covenant.org/) Community Code of Conduct.
 - Take the time to get things right. Pull Requests (PR) almost always require additional improvements to meet the bar for quality. Be very strict about quality. This usually takes several commits on top of the original PR.
 - Update documentation where necessary, write documentation when required. Use this repositories' wiki when targeting users / players of this game. Write green code or text files when targeting other developers.
 
 ## Requirements for code contributions to master branch
-- try to only contribute working code, no dead code, no "soon to be used" code and no "will fix it soon" code
-- for C# code, follow the usual C# / .Net coding style rules from Microsoft and Unity
+- Try to only contribute working code, no dead code, no "soon to be used" code and no "will fix it soon" code
+- No huge methods, try to reduce complexity, write readable code -> see [Clean Code Cheat Sheet](https://www.bbv.ch/images/bbv/pdf/downloads/V2_Clean_Code_V3.pdf)
+- When copying others peoples/projects code, check licenses
+
+## Code Style 
+The code style is configured in a file [`.editorconfig`](https://EditorConfig.org). Thus, when using Visual Studio Code (with OmniSharp) it should show warnings on code style violations. Furthermore, code formatting is configured to be done on save
```

Comment:
From what I know, Unity ships with MonoDevelop.
Visual Studio is Gigabytes of resources and heavyweight (e.g startup). I prefer vscode because it has all I need, is lightweight and cross-platform.

IMO people should be free to choose their favorite tools as long as the guidelines are followed. I provided the config for what I use. Feel free to find a suited place for it.

I dont know codacity.